alacritty_terminal/src/ansi.rs

```diff
@@ -1015,6 +1021,8 @@ where
             },
             ('S', None) => handler.scroll_up(Line(arg_or_default!(idx: 0, default: 1) as usize)),
             ('t', None) => match arg_or_default!(idx: 0, default: 1) as usize {
+                14 => handler.window_size_pixels(writer),
```

Comment:
We probably shouldn't return the text area size when the outer window size is requested.